### PR TITLE
5.4.2 version history

### PR DESCRIPTION
**Pull request by hflynn**
# What this PR does

Adds the version history for the 5.4.2 release.

# Testing this PR

Proofread. To check formatting for the docs, you'll need to manually copy the content into the matching docs page and build the sphinx docs.

# Related reading

See https://trello.com/c/ENudeoWD/30-version-history-pr and the 5.4.2 milestones on this repo & OME docs repo

 * [x] public_user
 * [x] developers section
 * [x] admin section
 * [x] scripts
 * [ ] other repos?

### CODE REVIEW

**Comment by hflynn** [comment]
initial commit should cover all the merged PRs and the contents of https://github.com/openmicroscopy/openmicroscopy/pull/5616 (next_infra re-integration) but does not yet cover the still open PRs on this repo or the expected extra doc PR for production hardening (https://trello.com/c/iTfw8nrD/57-production-hardening-page cc @sbesson )

File: history.txt [hunk 1]
```diff
@@ -5,6 +5,55 @@
...
+
+Improvements include:
+
+-  made projecting images belonging to another user only possible for users
+   with the required permissions to save the new images
+-  added documentation on a complete workflow for publishing data from
+   OMERO.server
+-  added references to the new OMERO pyramid format documentation (within the
+   OME Data Model and File Formats documentation)
+-  updated swingx library version used by OMERO.insight to stop insight-ij
```
Comment (jburel) [inline history.txt]: SwingX

File: history.txt [hunk 2]
```diff
@@ -5,6 +5,55 @@
...
+   plugin crashing in Fiji
+-  fixed issue with polygon and polyline ROIs created in OMERO.iviewer and
```
Comment (jburel) [inline history.txt]: not included in 5.4.2

Comment (hflynn) [reply history.txt]: Ah, i missed it was merged into workflows - it's labelled as milestone 5.4.2

Comment (joshmoore) [reply history.txt]: My fault. Like I said, I'll cross check ASAP.

Comment (jburel) [reply history.txt]: it was initially but I removed the label a while back

**Comment by jburel** [comment]
maybe for dev section

* re-organize python testing library to allow testing of external cli-plugin

**Comment by joshmoore** [comment]
@hflynn : pushed a commit for the first section. Next will be developers section unless you have any edits first. Primary outstanding addition is public user if it gets merged.

**Comment by hflynn** [comment]
@joshmoore edits look good, go ahead with the developer section, I'll start prepping the website PR

File: history.txt [hunk 3]
```diff
@@ -33,21 +33,20 @@ Sysadmin changes include:
...
+-  extended Python and Java examples to include Map Annotations and histograms
 -  added methods for updating OMERO.tables
 -  Java Gateway fixes for sessions and rendering
--  extended Python and Java examples to include Map Annotations and histograms
+-  fixed retrieval of Plate thumbnail URLs
 -  improved 'Editing OMERO.web' documentation
 -  improved Slice documentation for API deprecations
--  fixed retrieval of Plate thumbnail URLs
+-  substantial effort to make third-party repositories easily testable;
+   see `omero-test-infra <https://github.com/openmicroscopy/omero-test-infra>`
```
Comment (hflynn) [inline history.txt]: you're missing the '_' at the end of link here - that'll fail the doc build when copied across I think

**Comment by joshmoore** [comment]
Added public user and extending cli (merging code PRs now)

cc: @jburel @dominikl @hflynn @will-moore @rgozim

**Comment by hflynn** [comment]
Latest commit looks good to me, updating the website announcement version to match now

**Comment by hflynn** [comment]
https://github.com/openmicroscopy/www.openmicroscopy.org/pull/180 updated to current state of this PR now

**Comment by joshmoore** [comment]
General sign of in ... #general. Thanks all. Off to the release.

**Pull request merged**